parse/grammar/KuneiformParser.g4

```diff
@@ -191,6 +224,47 @@ common_table_expression:
     identifier (LPAREN (identifier (COMMA identifier)*)? RPAREN)? AS LPAREN select_statement RPAREN
 ;
 
+create_table_statement:
+    CREATE TABLE (IF NOT EXISTS)? name=identifier
+    LPAREN
+    (table_column_def | table_constraint_def | table_index_def)
+    (COMMA  (table_column_def | table_constraint_def | table_index_def))*
+    RPAREN
+;
```

Comment:
drop table?

Reply:
ah shoot...  Will add it now

Reply:
hmm, do you want to add `drop table` in our DDL docs? Or do I just use syntax from https://www.postgresql.org/docs/current/sql-droptable.html ?

Reply:
Yeah we should add drop ddl.

Reply:
Just base it on postgres's tho